### PR TITLE
Update the contributing guide

### PR DESCRIPTION
**Pull request by dingyin**
Add Contributor Workflow and format of commit message section

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -19,6 +19,20 @@ If you have any questions, contact us on our [Slack Channel](https://mizar-group
...
+Please do not ever hesitate to ask a question or send a pull request.
+
+This is a rough outline of what a contributor's workflow looks like:
+
+- Create a topic branch from where to base the contribution. This is usually master.
+- Make commits of logical units.
+- Make sure commit messages are in the proper format (see below).
+- Push changes in a topic branch to a personal fork of the repository.
+- Submit a pull request to [mizar](https://github.com/futurewei-cloud/mizar).
+- The PR must receive an approval from two maintainers.
```
Comment (haibinxie) [inline CONTRIBUTING.md]: Minor: receive approval from two maintainers.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -40,7 +54,35 @@ Each directory includes a module.mk files that are referenced by the main Makefi
 ### Commits
 
 It is not enough that we have a header for the commits. The commit
-needs to have one paragraph describing what it does.  We shall avoid Multi-pages commits as much as possible, but we still understand the needs to do otherwise.
+needs to have one paragraph describing what it does.  We shall avoid Multi-pages commits as much as possible, but we still understand the needs to do otherwise. The format of commit message is explained below.
```
Comment (er1cthe0ne) [inline CONTRIBUTING.md]: by "Multi-pages commits", do you mean Multi-pages comments?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -40,7 +54,35 @@ Each directory includes a module.mk files that are referenced by the main Makefi
...
+
+### Format of the commit message
+
+We follow a rough convention for commit messages that is designed to answer two questions: what changed and why.
+The subject line should feature the what and the body of the commit should describe the why.
+
+```
+scripts: add test codes for agent
+
+this add some unit test codes to imporve code coverage for agent
```
Comment (er1cthe0ne) [inline CONTRIBUTING.md]: typo: imporve->improve.

File: CONTRIBUTING.md (hunk 1 above)
Comment (zasherif) [reply CONTRIBUTING.md]: As agreed, we will change to two members and one maintainer.

**Pull request merged**